health-sdk/sdk/src/main/java/net/gini/android/health/sdk/review/model/PaymentDetails.kt

```diff
@@ -17,8 +17,10 @@ data class PaymentDetails(
 internal fun ExtractionsContainer.toPaymentDetails() = PaymentDetails(
     recipient = specificExtractions["paymentRecipient"]?.value ?: "",
     iban = specificExtractions["iban"]?.value ?: "",
-    amount = specificExtractions["amountToPay"]?.value?.toAmount() ?: "",
-    purpose = specificExtractions["paymentPurpose"]?.value ?: "",
+//    amount = specificExtractions["amountToPay"]?.value?.toAmount() ?: "",
```

Comment:
Was it a temporary change? 😊

Reply:
Yes, and committed by accident. Also this problem was caught by failing tests as well 🎉 ! #testsftw